public/API/API_GetUserWantToPlayList.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ *  API_GetUserWantToPlayList - returns a list of GameIDs that a user has saved on their WantToPlayList
+ *    u : username
+ *    o : offset - number of entries to skip (default: 0)
+ *    c : count - number of entries to return (default: 100, max: 500)
+ *  int         Count                       number of want to play game records returned in the response
+ *  int         Total                       number of want to play game records the user actually has overall
+ *  array       Results
+ *   object      [value]
+ *    int        ID                         unique identifier of the game
+ *    string     Title                      name of the game
+ *    int        ConsoleID                  unique identifier of the console associated to the game
+ *    string     ConsoleName                name of the console associated to the game
```

Comment:
This isn't included in the response.

Reply:
Good catch, removed. I was hoping to include it in the response, but didn't have access on the tables I am querying.